src/meta/meta_bulk_load_service.cpp

```diff
@@ -1408,11 +1462,10 @@ void bulk_load_service::on_query_bulk_load_status(query_bulk_load_rpc rpc)
     }
 
     if (!app->is_bulk_loading) {
-        auto hint_msg = fmt::format("app({}) is not during bulk load", app_name);
+        auto hint_msg = fmt::format("app({}) is not during bulk load, return last time result", app_name);
```

Comment:
There are two cases:
1. table finish last round bulk load, it will return last time result
2. table has never executed bulk load before, it don't have last time result, it should return some error, at least should be different from case1

I suggest updating code to consider case 2 .